### ISSUE TITLE
Error: Product Identifiers don't match

### ISSUE THREAD

**Issue by jonatha968** [maciekish/iReSign#35]
I am trying to resign a .ipa currently signed by our developer with out enterprise profile and cert. 

![iresign](https://cloud.githubusercontent.com/assets/8009337/4910644/300bcbea-6482-11e4-9fb7-1ff2c8c54733.png)

**Comment by zonabi** [comment]
did u figure it out? having same issue

**Comment by jonatha968** [comment]
I ended up using a different product named Airsign.
�
Jonathan Smith
Sr. Systems Engineer
Gilead Sciences, Inc.
Foster City, CA. 94404
Tel:  650-372-7094
Cell: 650-302-6985
E:  Jonathan.Smith@gilead.com<mailto:Jonathan.Smith@gilead.com>

**Comment by zonabi** [comment]
is it legitimate to submit an app to Apple App store after resign-ing it with one of these tools?

**Comment by Caddington** [comment]
@zonabi Yes it is, so long as you're signing it using your own provisioning profile and certificate.  It does not cause any issues with the app publishing process (binary upload to iTunesConnect, submission, review, and release).

**Comment by willbin** [comment]
Met the same problem.

And solved with changing to `if(1)` here. 

`iReSignAppDelegate.m`

```
if (1)
// if (identifierOK)
{
    NSLog(@"Provisioning completed.");
    [statusLabel setStringValue:@"Provisioning completed"];
    [self doEntitlementsFixing];
} else {
    [self showAlertOfKind:NSCriticalAlertStyle WithTitle:@"Error" AndMessage:@"Product identifiers don't match"];
    [self enableControls];
    [statusLabel setStringValue:@"Ready"];
}
```

**Comment by ConorBaumgartDHX** [comment]
@willbin How did you get this change to take effect in the app?

**Comment by malokuS** [comment]
Make sure to have a correct provisioning profile!
